example_data/example_seg_simple.yml

```diff
@@ -12,8 +12,3 @@ segments:
   segment_id: example_c2
   start: 2020-01-22 15:15:00
   end: 2020-01-22 15:25:00
-- kinds:
```

Comment:
Is there any reason to remove this? I added it, so that we can check whether we appropriately catch if there is an 'empty circle'. I'd rather include a catch for this instead of removing the example

Reply:
The code fails when there is an empty circle. Removing it is probably not the best solution, then maybe we should add a way to deal with the empty circle.